site-packages/integralstor_gridcell/grid_ops.py

```diff
@@ -980,6 +984,33 @@ def add_a_gridcell_to_storage_pool(si, hostname):
             raise Exception(','.join(error_list))
         print 'Mounted admin volume.'
 
+        # Check if the admin vol mount actually succeded by reading a file from the admin volume
+        # Tries 3 times. Bail out if unsuccessfull
+        print "Check if mounted admin vol is accessible"
+        if rc is True:
+            iter = 3
+            while (iter):
+                iter -= 1
+                is_mount_ok = True
+                ret = client.cmd(hostname, 'file.file_exists', [
+                    '/opt/integralstor/integralstor_gridcell/config/lock/assert_admin_vol_mount'])
```

Comment:
Can we use the generic config dir call from config.py so we avoid full dir references in the code?